SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentTestUtils.h

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/EntityComponentTypes.h"
+
+namespace SpatialGDK
+{
+
+static const Schema_FieldId EventId = 1;
```

Comment:
These need to be named like namespace level constants (I think we use macro-style all caps).
They should also probably be named to make it clear that they are for a specific test component.
The the worker wrapper branch this was in another namespace but I think originally this was just me being lazy.

Reply:
Updated constant names everywhere, added a namespace in Utils header.